packages/falcon-server-env/src/cache/Cache.ts

```diff
@@ -5,7 +5,7 @@ export type SetCacheOptions = {
   ttl?: number;
 };
 
-export type CacheResult = string | undefined;
+export type CacheResult = string | any | undefined;
```

Comment:
`any` ? so for what we have `string` and `undefined`?

```suggestion
export type CacheResult = any;
```

Reply:
true that :)